src/main.c

```diff
@@ -156,12 +185,24 @@ typedef struct {
 static Model model;
 static Model *g = &model;
 
-int chunked(float x) {
+/// As a part of [Issue #59]:https://github.com/Team-10-But-Better/Craft/issues/59
+/// The following function initializes the player's SurvivalStatus
+void initializeSurvivalStatus()
```

Comment:
Maybe consider reducing this down to "initSurvivalStat()", its quite a long function name right now